src/avram/database.cr

```diff
@@ -263,7 +263,7 @@ abstract class Avram::Database
     def truncate
       return if table_names.empty?
 
-      statement = ("TRUNCATE TABLE #{table_names.map { |name| name }.join(", ")} RESTART IDENTITY CASCADE;")
```

Comment:
From https://www.postgresql.org/docs/current/sql-truncate.html :
> RESTART IDENTITY
Automatically restart sequences owned by columns of the truncated table(s).

Could we keep the original behavior maybe as a method argument ?
`def truncate(restart_identity = true)`

Reply:
oh, that's a good point :thinking: This all happens in a few different areas

Truncating the database
https://github.com/luckyframework/avram/blob/df3c20f195aa6f844372fa1089ec71fe55f47399/src/avram/database.cr#L43-L46

Truncating a table
https://github.com/luckyframework/avram/blob/df3c20f195aa6f844372fa1089ec71fe55f47399/src/avram/queryable.cr#L54-L61

Maybe we need to add for both?

Reply:
That's probably a better idea. It maintains the original behaviour by default, just in case someone depended on it.

Reply:
I could push up subsequent commits with the requested changes, if you're up for it?

> Truncating a table

Should this be a separate PR, or bundling it with this is OK?

Reply:
I'm ok with it being bundled. Once this is all merged in, I'll do a quick Avram release.